docs/ansible/roles/haproxy/getting-started.rst

```diff
@@ -0,0 +1,42 @@
+Getting started
+===============
+
+.. contents::
+   :local:
+
+
```

Comment:
A __Default Configuration__ section should give a brief overview of the state the application will be in if deployed with defaults, especially:
* choices that were made that might differ from its out-of-the-box defaults
* things the end user _must_ pay attention to before deploying to production